src/js/demo/components/BugReport.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function BugReport(props) {
+    return (
+        <div className="alert report-message-alert alert-danger">
+            <div>
+                {props.message}
+            </div>
+            <div>
+                Please submit a bug report at:
+            </div>
+            <div>
+                <a href="https://github.com/eslint/eslint/issues">https://github.com/eslint/eslint/issues</a>
```

Comment:
Should we point it to "New Issue" page directly instead of issues list?

Reply:
That's a good idea! 

Only, if the user isn't logged in (e.g. they don't have a GitHub account yet), they'll see only the login page? What's the usual practice for 'report issue' buttons/links?

Reply:
Yes, they'll be redirected to login page.